tests/Unit/Tpay/Resolver/TpayTransactionChannelResolverTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\CommerceWeavers\SyliusTpayPlugin\Unit\Tpay\Resolver;
+
+use CommerceWeavers\SyliusTpayPlugin\Payum\Request\Api\GetTpayTransactionsChannels;
+use CommerceWeavers\SyliusTpayPlugin\Tpay\Resolver\TpayTransactionChannelResolver;
+use Payum\Core\GatewayInterface;
+use Payum\Core\Payum;
+use PHPUnit\Framework\TestCase;
+use Prophecy\PhpUnit\ProphecyTrait;
+use Prophecy\Prophecy\ObjectProphecy;
+
+final class TpayTransactionChannelResolverTest extends TestCase
+{
+    use ProphecyTrait;
+
+    private Payum|ObjectProphecy $payum;
+
+    protected function setUp(): void
+    {
+        $this->payum = $this->prophesize(Payum::class);
+    }
+
+    public function test_it_resolved_indexed_tpay_transaction_channels(): void
+    {
+        $this->markTestSkipped('Issue with GetTpayTransactionsChannels');
```

Comment:
I cannot make it to work as this GetTpayTransactionsChannels throws:

`Typed property GetTpayTransactionsChannels::$result must not be accessed before initialization`